public/catalog/v1.0.0/openapi.json

```diff
@@ -697,6 +704,54 @@
         }
       }
     },
+    "/portfolio_items/{portfolio_item_id}/undelete": {
+      "post": {
+        "tags": [
+          "PortfolioItem"
+        ],
+        "summary": "UnDelete a specified Portfolio Item",
```

Comment:
Small nit, but this appears to be the only place we use camel case for the term 'undelete'. Possibly just capitalize the 'U'.